tests/ui/pages/data_entity/overview.page.ts

```diff
@@ -96,6 +92,10 @@ export default class OverviewPage extends DataEntityPage {
     return new Radio(this.page, SELECTORS.addCustomMetadataTrueRadioButton);
   }
 
+  async getField(inputName: InputName, name: string) {
+    await this[`input${inputName}`].fill(name);
+  }
+
```

Comment:
I previously had a comment on this method:
https://github.com/opendatadiscovery/odd-platform/pull/1120#discussion_r1036116636
And marked that thread as resolved (Probably by mistake).

Could u pls have a look at this comment?